compatible/prepare_backup.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright 2020 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+# update tags
+git fetch --tags
+
+# get latest 3 version clusters
+TAGS=$(git for-each-ref --sort=creatordate  refs/tags | awk -F '/' '{print $3}' | tail -n3)
```

Comment:
we need to test the new version BR can restore old version backups to old version cluster. so v5.0.0-nightly also can restore on release-4.0 cluster.

Reply:
@3pointer then in the far future, *on release-4.0 branch with BR = e.g. v4.0.20*, it will only test against (say) v5.0.8, v5.0.9, v5.0.10?

Reply:
🤔️，yes, if we had such tags in order [v4.0.19, v5.0.8, v5.0.9, v5.0.10, v4.0.20]
and before v5.0.10 published. we already tested [v4.0.19, v5.0.8, v5.0.9] can restore normally in v4.0.20

Reply:
With v5.x we may accidentally backup a table with new feature (e.g. clustered primary index) that causes the test to always fail on v4.x. 

----

Also PTAL @IANTHEREAL .

Basically, the compatibility test currently checks against the 3 most recent releases, regardless of major or minor version. Does this meet the expectation?

Reply:
If `the test to always fail on v4.x`, I think that's what we want. because we need to know the exact failed case so that we can avoid the failure. (we may find another feature just like clustered index, which makes BR failed, so we can choose adjust the test or update documents then. )